.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        node: [ '12', '10' ]
+    name: Node ${{ matrix.node }} (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: install dependencies
+        run: npm install --ignore-engines
+      - name: install codecov
+        run: npm install -g codecov
+      - name: lint code
+        run: npm run lint
+      - name: run tests
+        run: npm test
+      - name: get code coverage report
+        run: codecov
+      - name: do semantic release
+        run: npm run semantic-release
```

Comment:
@DanielRuf could we add here the tokens that the action will need in order to release?

```
env:
  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
  NPM_TOKEN: ${{secrets.NPM_TOKEN}}
```

Reply:
I did not see this code in the Travis config but sure, see https://help.github.com/en/articles/virtual-environments-for-github-actions#environment-variables and https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables

Reply:
You should be able to push to my branch but I think you meantthat I add this. Will do so now.

Reply:
Yep. If you can add it to the PR then we'll be all set because I already configured the needed token secrets for this repo.

Reply:
Done.